### ISSUE TITLE
Import project into Bnd Workspace

### ISSUE THREAD

**Issue by diderotnet** [bndtools/bndtools#1400]
When importing a project into Eclipse using the EnRoute environment, if the copy checkbox is set the project ends up in the Eclipse workspace.
At present the project dir and contents have to be manually copied to the bnd/git workspace.
Bndtools	3.2.0.REL-20160517-162512-g3a83c33
Eclipse IDE for Java Developers 4.5.2.20160218-0600	epp.package.java
Win 10 64Bit

**Comment by gamerson** [comment]
I think we can address this along with other "import" use-cases by using the new Neon Smart Import framework.  https://wiki.eclipse.org/E4/UI/Smart_Import

**Comment by bjhargrave** [comment]
But I guess that would make bndtools require Neon as the base version of Eclipse. So we need to make that decision prior to writing a smart importer.

**Comment by gamerson** [comment]
agree, @bjhargrave could we add smart importer to a new bundle?  and that bundle can be marked as optional so it bndtools plugin will still resolve in non-neon installs?

**Comment by bjhargrave** [comment]
I think the problem is that we will need to build bndtools against Neon (we now build against Luna). Once we make that change, we risk creeping dependencies beyond Luna.

**Comment by gamerson** [comment]
hmm, i see, its hard to have both repos setup in bndtools build.
